ref-exchange/src/rated_swap/README.md

```diff
@@ -0,0 +1,29 @@
+### RatedSwapPool for stNEAR/wNEAR pool
```

Comment:
As it is not a full completed version, please make a new PR to the `feature_rated_stable_pool` branch. More review and test work would be managed on that branch before merging into `main`.